runtime/vm/jvminit.c

```diff
@@ -1682,10 +1682,10 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				IDATA argIndex9 = 0;
 				BOOLEAN sharedClassDisabled = FALSE;
 
-#if defined(J9VM_ARCH_X86)
+#if !defined(J9VM_ARCH_POWER)
```

Comment:
I would use explicit list of supported platforms rather than use "all except Power". What about ARM, RiscV, ZTPF etc?